code/src/components/Fondness.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+const Fondness = ({ fondness, setFondness }) => {
```

Comment:
I really liked this feature! Will go back and be inspired for future projects :)